insights/parsers/freeradius.py

```diff
@@ -0,0 +1,53 @@
+"""
+FreeRadius - Freeradius configuration files
+===========================================
+
+FreeRadius is a parser for the freeradius configuration files usually in
+``/etc/raddb``.
+Files ending with ``.rpmnew`` are excluded because they are backups made when
+ugprading but customer modified the configuration files, so are not used by the
+service.
+
+Because this parser reads multiple files, the files are stored as a list within
+the parser and need to be iterated through in order to find specific
+configuration files.
+
+Examples:
+
+    >>> for cfgfile in shared[FreeRadius]: # Parser contains list of all configuration files
+    ...     print "Path:", cfgfile.file_path
+    ...
+    Path: etc/raddb/radiusd.conf
+    Path: etc/raddb/proxy.conf
+    Path: etc/raddb/policy.d/filter
+
+"""
+
+from .. import Parser, parser
+from . import SkipException
+
+
+@parser("freeradius")
+class FreeRadius(Parser):
```

Comment:
The parser doesn't provide an API above the raw data of the datasource behind it.

The objects returned by datasources already contain file path information and contents.

Ignoring files can be done at collection time by passing a regular expression to `glob_file` with its `ignore` keyword (see `glob_file`'s entry in the [Insights Core Tutorial](https://github.com/RedHatInsights/insights-core/blob/master/notebooks/Insights%20Core%20Tutorial.ipynb)), and basic line cleansing can be handled with [get_active_lines](https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/__init__.py#L30).

Parsers and combiners should provide a higher level API to rules and other components so they don't have to deal with parsing individual lines themselves.